src/cli/options.ts

```diff
@@ -46,6 +46,14 @@ export function withStateOption<T>(
   });
 }
 
+export function withIdOption<T>(yargs: Argv<T>): Argv<T & { id?: string[] }> {
+  return option(yargs, "id", {
+    string: true,
+    type: "array",
```

Comment:
This makes the help menu report the type as "array" when it should be string. Does it work without this line?

Reply:
Yes, but then it only is able to take a single string as an argument each time the flag is specified. 

I could very well be in the minority, but I found the behavior of the ignore flag surprising as it seems to only be able to take a single path per use of the flag

So this works:

`--ignore ./path-1 --ignore ./path-2`

Ignore is populated with: `["./path-1", "./path-2"]`

However this was unexpected:

`--ignore ./path-1 ./path-2`

Ignore is populated with: `"./path-1"`

Where as with the "array" type both syntaxes above work to pass multiple arguments. I think array may be a better type for `--id` and `--ignore`, but I could very well be wrong about this. That is my thinking though on why I support keeping array.

Reply:
Doesn't your example run the risk of having `./path-2` be ambiguous (to the user) between an additional ignore argument and the actual source argument (which takes no flag)?

Reply:
Good point, I hadn't though of that. I'll change the behavior to be the same as the current `--ignore` flag (with type as string).